hack/deploy_csv.yaml

```diff
@@ -250,7 +250,8 @@ data:
           * Red Hat CodeReady Workspaces
           * OpenShift Pipelines
           ## Installation
-          CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and the developer tools get installed in a single namespace.
+          CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and OpenShift Pipelines get installed in single `openshift-operators` namespace and Red Hat CodeReady Workspaces in single `toolchain-che` namespace.
+          ###### _Warning: This operator is not supposed to be used in OKD - by installing it there you will get unexpected errors. Please, use with OpenShift only._
```

Comment:
```suggestion
          ###### _Warning: This operator should be installed in OpenShift Container Platform clusters only. Installing on OKD or other platforms will result in errors._
```

Reply:
same as the previous case - changed before checking your comment d58c62c